Develop

moiseenko-e opened the pull request.

moiseenko-e committed:
first commit

moiseenko-e committed:
pom

nikolay-977 commented on src/README.md:
```diff
@@ -0,0 +1,7 @@
+Тестирование API сервиса [Яндекс.Самокат](qa-scooter.praktikum-services.ru/docs/):
+
```
Можно улучшить: хорошей практикой является добавление README.md с описанием проекта, которое включает название, технологии в проекте (например, Java 11, JUnit 4.13.2, maven 3.9.0 и т.д.), настройка (если необходима) и как запускать (mvn clean test). Как оформлять README.md можно ознакомиться в статье https://dzen.ru/a/YbzudflBPE1IDJF-

moiseenko-e replied:
Добавила

nikolay-977 commented on src/main/java/ru/praktikum/Courier.java:
```diff
@@ -0,0 +1,49 @@
+package ru.praktikum;
```
Можно улучшить: Хорошая практика - размещать классы в пакетах, объединяя их по функциональности (классы с запросами в src.main.java.ru.praktikum.burgers.api.client, классы с описанием модели данных в src.main.java.ru.praktikum.burgers.api.model, а в вспомогательные классы в src.main.java.ru.praktikum.burgers.api.util) либо по сущностям (src.main.java.ru.praktikum.burgers.api.courier, src.main.java.ru.praktikum.burgers.api.order)

moiseenko-e replied:
Обязательно учту при работе с дипломом, большое спасибо

nikolay-977 commented on src/test/java/ru/praktikum/CreateCourierTests.java:
```diff
@@ -0,0 +1,74 @@
+import org.junit.Test;
+import static org.hamcrest.Matchers.equalTo;
+
+// "https://qa-scooter.praktikum-services.ru"
```
Необходимо исправить: удали лишний комментированный код

moiseenko-e replied:
Удалила

nikolay-977 commented on src/test/java/ru/praktikum/OrderListTests.java:
```diff
@@ -0,0 +1,31 @@
+                .header("Content-type", "application/json")
+                .get(API_ORDER);
+        response.then().log().all();
+        MatcherAssert.assertThat("orders", notNullValue());
```
Необходимо исправить: Проверки в тестовых методах должны быть в одном стиле. Когда код написан в одном стиле, это улучшает читаемость кода, сокращает время локализации дефекта и упрощает его поддержку. Не следует в тестовых методах одновременно использовать проверки из разных фреймворков org.hamcrest.MatcherAssert(метод assertThat) и RestAssured. В этих фреймворках отличается формат сообщений об ошибках в случае фейла, что не очень удобно при локализации дефектов.

moiseenko-e replied:
Исправила, но не совсем поняла, делала строго по памятке в курсе, но может не так поняла, поизучаю

nikolay-977 reviewed (commented):
Можно улучшить: хорошей практикой является добавление README.md с описанием проекта, которое включает название, технологии в проекте (например, Java 11, JUnit 4.13.2, maven 3.9.0 и т.д.), настройка (если необходима) и как запускать (mvn clean test). Как оформлять README.md можно ознакомиться в статье https://dzen.ru/a/YbzudflBPE1IDJF-

moiseenko-e committed:
Fixes

Исправлено:
Удален закоммиченный текст
Исправлен тест в классе OrderListTests
Улучшено:
Исправлен файл README.md
